### ISSUE TITLE
[BUG] - Accounts settings gone

### ISSUE THREAD

**Issue by MariuzM** [Atlas-OS/Atlas#1248]
### Before continuing...

- [X] I agree to follow Atlas' [Code of Conduct](https://github.com/Atlas-OS/.github/blob/main/profile/CODE_OF_CONDUCT.md)
- [X] I have searched our [issue tracker](https://github.com/Atlas-OS/Atlas/issues?q=is%3Aissue) to verify that the issue hasn't been reported already
- [X] I have searched the [Atlas documentation](https://docs.atlasos.net) beforehand for a solution
- [X] I am on the latest version of Atlas, and didn't do an unsupported upgrade

### Description

I just noticed today when for some reason windows asked me to create password where before i never used password and just auto login. 
But then i wanted to remove it and i could not find Accounts settings anywhere. My account is local is that the reason, I'm sure I’ve seen it before. 

I understand there is new Playbook update but i can't use it without reinstating windows right? 

basically how can i get accounts settings page pack? 

### Steps to reproduce

Open Settings > no Accounts page

### Expected behavior

Accounts should be there

### Actual behavior

No Accounts... who is creating these questions? :D 

### Atlas Edition

Atlas for Windows 11 23H2

### Desktop information

N/A

**Comment by he3als** [comment]
See https://github.com/Atlas-OS/Atlas/issues/1140#issuecomment-2212516706, this is an issue with v0.4 and has been fixed in v0.4.1, the latest version as of writing this.

As a side note, if you want to know more about upgrading: https://docs.atlasos.net/general-faq/updating-atlas/

**Comment by MariuzM** [comment]
@he3als thank you it does specify `$ver = 'v0.3.3'` this version i used 0.4.0, should i change?

**Comment by he3als** [comment]
> @he3als thank you it does specify `$ver = 'v0.3.3'` this version i used 0.4.0, should i change? 

That's the ViVeTool version, not Atlas version, so it should work without changing anything

**Comment by shadowking1443** [comment]
> See [#1140 (comment)](https://github.com/Atlas-OS/Atlas/issues/1140#issuecomment-2212516706), this is an issue with v0.4 and has been fixed in v0.4.1, the latest version as of writing this.
> 
> As a side note, if you want to know more about upgrading: https://docs.atlasos.net/general-faq/updating-atlas/

Hi @he3als - is there a way to fix this while still on version 0.4.0, such as a power shell command? I can't upgrade yet as it will require a new installation, which isn't feasible at this time
